src/test/java/gov/healthit/chpl/aqa/stepDefinitions/SurveillanceReportSteps.java

```diff
@@ -111,5 +111,43 @@ public void iEditQuarterlySurveillanceReport(final String acbNameYearQuarter) {
     public void navigateToSurveillanceReportsPageAsAnonymousUser() {
         getDriver().get(getUrl() + "#/surveillance/reporting");
     }
-}
 
+    @And("^I click open Listings with relevant surveillance accordion$")
+    public void openListingsWithRelevantSurveillanceAccordion() {
+        WebElement button = SurveillanceReportPage.listingsWithRelevantSurveillanceAccordion(getDriver());
+        ((JavascriptExecutor) getDriver()).executeScript("arguments[0].scrollIntoView(true);", button);
+        ((JavascriptExecutor) getDriver()).executeScript("arguments[0].click();", button);
+    }
+
+    @And("^I click View listing surveillance data button for CHPL ID \"([^\"]*)\"$")
+    public void clickViewListingSurveillanceDataButton(final String chplID) {
+        SurveillanceReportPage.viewListingSurveillanceDataButton(getDriver(), chplID).click();
+    }
+
+    @And("^I click Edit surveillance data button for Surveillance Id \"([^\"]*)\"$")
+    public void clickEditSurveillanceDataButton(final String friendlySurvId) {
+        SurveillanceReportPage.editSurveillanceDataButton(getDriver(), friendlySurvId).click();
+    }
+
+    @And("^I enter surveillance data in fields on form: \"([^\"]*)\" \"([^\"]*)\" \"([^\"]*)\" \"([^\"]*)\" \"([^\"]*)\" \"([^\"]*)\" \"([^\"]*)\"$")
+    public void enterSurveillanceData(final String outcome, final String processType, final String grounds,
+            final String causes, final String additionalCosts, final String limitationsEvaluation,
+            final String completedCapVerification) {
+        getDriver().manage().window().maximize();
+        SurveillanceReportPage.surveillanceOutcomeDropdown(getDriver()).sendKeys(outcome);
+        SurveillanceReportPage.surveillanceProcessTypeDropdown(getDriver()).sendKeys(processType);
+        SurveillanceReportPage.groundsForInitiatingInput(getDriver()).sendKeys(getCurrentDate());
```

Comment:
It looks like most of the parameters aren't being used. Is that on purpose?

Reply:
yes. I parameterized the test in case we want to expand the test to test with different inputs but with current testing practice to ensure that form has latest saved update, like we use on other forms, date is used as input.